src/components/layouts/Submenu.tsx

```diff
@@ -1,16 +1,33 @@
 import React from 'react';
 
-export default class Submenu extends React.Component<{
+interface SubmenuPropsTypes {
   module: string,
   submodule: string,
   getAddress: (sAddress: string) => void
-}> {
-  state = {
-    module: '',
-    urlSlot: '',
-    methodSlot: '',
-    routeSlot: '',
-    paramSlot: ''
+}
+
+interface SubmenuStateTypes {
+  module: string,
+  urlSlot: string,
+  methodSlot: string,
+  routeSlot: string,
+  paramSlot: string
+}
+
```

Comment:
This is a good approach. Please implement the same for other modules like `Benchmarks.tsx`, `Ping.tsx`.

Reply:
Okay, will do and push

Reply:
Done